rfcs/20220128-f64-data_type/Readme.md

```diff
@@ -0,0 +1,72 @@
+# RFC: f64 convolution support
+
+## Introduction & Motivation
+
+Some deep learning applications require higher numerical accuracy than traditionally 
+supported 32-bit floating points, especially for training phase. This RFC addresses
+this requirement by introducing 64-bit, double precision float convolution support in oneDNN.
+We propose adding f64 data type support for convolution on the GPU. Since `reorder` is used to support convolution with different tensor formats, fp64 support will be added to this primitive as well. Other primitives, or engines are not impacted at this point; although library infrastrcture is modified to recognize double-precision floating-point data type.
```

Comment:
I agree, we should clarify that f64 support is enabled only on platforms that have HW support for it.
Also, could you clarify that we would need only f32 <-> f64 reorder? (mostly for testing purposes)

Reply:
> Also, could you clarify that we would need only f32 <-> f64 reorder? (mostly for testing purposes)
For now, yes. Since benchdnn uses f32 for reference compute. If, in the future however, we wanted to compute f64 reference compute, we would need f64 <-> f64 as well.